docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -140,7 +148,7 @@ You'll need to restart the Fleet instances after changing these settings.
 ### Advanced Configuration
```

Comment:
I realize this was already like this, but gonna suggest fixing the heading capitalization while we're here:
```suggestion
### Advanced configuration
```